examples/expo/src/constants/screenStyles.js

```diff
@@ -59,7 +59,7 @@ const screenStyles = StyleSheet.create({
   message: {
     color: colors.white,
     fontSize: constants.responsiveWidth(11),
-    lineHeight: 48,
+    lineHeight: constants.responsiveWidth(11) * 1.2,
```

Comment:
why this value? And why fontSize depends on width?

Reply:
or we can use system font scale? https://reactnative.dev/docs/pixelratio 🤔

Reply:
i just used current solution

Reply:
fixed